views/beer.hbs

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
you don't need to add any of the initial tags of an html for every hbs file. This only needs to be on the layout, every other file should only include what you have inside the body tag. Not even the body tag needs to be added.